packages/flutter_tools/test/general.shard/android/gradle_test.dart

```diff
@@ -2701,7 +2701,7 @@ plugin1=${plugin1.path}
       legacySettingsDotGradleFiles.readAsStringSync().split(';EOF').map<String>((String body) => body.trim()),
       contains(templateSettingsDotGradle.readAsStringSync().trim()),
     );
-  });
+  }, skip: true); // TODO(jonahwilliams): This is an integration test and should be moved to the integration shard.
```

Comment:
This TODO probably needs a github issue if we don't already have one.

Reply:
https://github.com/flutter/flutter/issues/69975